libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -58,6 +57,10 @@ public function prepareStorageBucketAndTable(
                 $processedSource,
                 $processedSource->getDestination(),
             );
+
+            $destinationTableInfo = $this->getDestinationTableInfoIfExists(
```

Comment:
Když to modifikuje strukturu tabulek, tak by se tu měl imho udělat  reload, aby `MappingStorageSources` obsahovalo informace a změněné tabulce.